gobreaker.go

```diff
@@ -18,6 +19,25 @@ const (
 	StateOpen
 )
 
+// ErrTooManyRequests is returned when the CB state is half open and the requests count is over the cb maxRequests
+var ErrTooManyRequests = errors.New("too many requests")
+
+// ErrCircuitOpen is returned when the CB state is open
+var ErrCircuitOpen = errors.New("circuit breaker is open")
+
+// Error is a generic error returned from the CB, wrapping the error and containing the CB name
+type Error struct {
```

Comment:
It's more convenient that CircuitBreaker errors can be compared with the exported errors directly:
```
_, err := cb.Execute(req)
if err == ErrTooManyRequests {
  ...
}
```

I will add Name method to CircuitBreaker so that you can get the name from the CircuitBreaker directly and Error struct gets unnecessary.
https://github.com/sony/gobreaker/commit/572391e561cb0a93d766397a9d6ce1424e843ba6